### ISSUE TITLE
Rename dataset field for 'use by others'

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#682]
As I'm almost always logged in, I only see now that anonymous users can only see the datasets they can apply for access to (right now: NGT, NGT-ns, EmblemsNL). Only when they have an account and are logged in, they see all datasets, irrespective of whether they can apply for access to them. Thus, they don't even get to the field that was intended for them in the dataset metadata page: "Conditions of use by others". (A misnomer, In fact, hence #683.) In this way, we don't advertise the datasets we have, nor the system as a possible location for new datasets by new users.
Therefore: please make sure that anonymous users can see all dataset pages, whether or not they can currently apply for access when they register, and whether or not they have public glosses in them. The latter is not really related to begin with, but the former does seem to be, there's a setting in the dataset admin that makes the three 'Dutch' datasets above visible: "Is public".
In other words (I'm writing this as I'm finding out what the situation is): we should keep using the existing setting "Is public" for sharing in the sense of allowing users to apply for access, but make the metadata for all datasets available to anonymous users at all times.

**Comment by susanodd** [comment]
Do you want the anonymous user to be able to see how many glosses are in the datasets?

**Comment by susanodd** [comment]
At the moment, the column to apply for access is only available for logged in users.
Make this column visible?
There actually isn't any place to click on anything in the Available Datasets list, only to see the Detail page, but not to try to gain access.

**Comment by ocrasborn** [comment]
Hi @susanodd , both suggestions are worth following up on, please go ahead and implement (quoting you to be sure we're on the same page):
* Maybe for anonymous users there can be a Request Access button on the Dataset Detail page?
Then they also need to create an account.
* Or maybe just a message telling them they need to create an account if the user viewing the details is anonymous.

**Comment by susanodd** [comment]
I'm working on this issue now.
I've made it so there's a button to request access if the user is anonymous.
One glitch, the current registration form only offers the choice of is_public datasets.
So this means that if the user has requested access to a dataset that is not public, it won't be offered.
At the moment, there are only 4 public datasets.
Easy solution is to only show this button for public datasets.

**Comment by susanodd** [comment]
![dataset-detail-ngt-anonymous](https://user-images.githubusercontent.com/24272544/114021950-8da41800-9871-11eb-9288-0bdd0349ba93.png)

![registration-preselect-ngt](https://user-images.githubusercontent.com/24272544/114021974-9399f900-9871-11eb-8b94-1e5cefd7d5f3.png)

**Comment by ocrasborn** [comment]
Hi @susanodd , what happens if users click that button? They should be required to create an account first, is that what happens?
In response to the second point: indeed, only show the button for public datasets!!

**Comment by susanodd** [comment]
Yes, that's the second screenshot, that's what's shown. (Here for NGT.)
I'll add code to check whether the user already exists if they start filling it in and already have an account.
Then it will just be a request access if needed.

**Comment by susanodd** [comment]
The second screenshot is the normal page for creating an account, but the dataset has been pre-selected to what they were just looking at when they clicked the button.

**Comment by ocrasborn** [comment]
That's very elegant from the user's perspective, the pre-selection! Great, @susanodd !

**Comment by susanodd** [comment]
This has been pushed to github branch admin_issues_739_682.
